enclosed/umbrel-app.yml

```
@@ -0,0 +1,32 @@
+manifestVersion: 1
+id: enclosed
+category: files
+name: Enclosed
+version: "1.6.9"
+tagline: Send private and secure notes
+description: >-
+  Enclosed is a minimalistic web application designed for sending private and secure notes. Whether you need to share sensitive information or just want a simple way to send encrypted messages, Enclosed provides a user-friendly interface with strong security features to ensure your data remains confidential.
+
+  The name Enclosed is inspired by the idea of keeping your notes safe and secure, like a sealed envelope.
+
+  
+  🛠️ SET-UP INSTRUCTIONS 🛠️
+
+  Enclosed relies on native browser crypto APIs to encrypt and decrypt notes with security and great performance. These APIs are only available in secure contexts (HTTPS).
+  If you are running the app in a non-secure context (HTTP), you will see a warning in the app.
+
+  You can set up https using the Cloudflare Tunnel app available in the app store.
+  Detailed instructions for configuring the tunnel can be found here: https://github.com/Radiokot/umbrel-cloudflared/wiki/How-to-set-up-Cloudflare-Tunnel-on-your-Umbrel
+releaseNotes: ""
+developer: Corentin Thomasset
+website: https://enclosed.cc/
+dependencies: []
+repo: https://github.com/CorentinTh/enclosed
+support: https://github.com/CorentinTh/enclosed/discussions
+port: 8787
```

This is the value to change. No other change should be necessary. For example:

```suggestion
port: 8788
```